schema/exampleSurveyJS.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
should we delete the schema folder?

Reply:
Yes, I think, it's not in use / outdated. What do you think, @ChristophWersal ?

Reply:
I removed it.